corehq/apps/cloudcare/static/cloudcare/js/formplayer/main.js

```diff
@@ -21,6 +21,7 @@ hqDefine("cloudcare/js/formplayer/main", function () {
             singleAppMode: initialPageData('single_app_mode'),
             environment: initialPageData('environment'),
             changeFormLanguage: toggles.toggleEnabled('CHANGE_FORM_LANGUAGE'),
+            enablePrintOption: toggles.toggleEnabled('PRINT_BUTTON_WITHIN_FORMS'),
```

Comment:
Nitpick: recognizing that you're following the existing pattern, what would you think of calling `toggleEnabled` directly in the places where it's used? It isn't an expensive call, and it would save passing around some flags.

Reply:
so many typos in the git commit message, but [5dae30d](https://github.com/dimagi/commcare-hq/pull/32641/commits/5dae30d2a0a4a515eb6fefa1681a448fd021c213)